src/dbt_osmosis/core/osmosis.py

```diff
@@ -849,7 +849,7 @@ def _run(self, unique_id, node, schema_map, force_inheritance=False):
             n_cols_added = 0
             n_cols_doc_inherited = 0
             n_cols_removed = 0
-            n_cols_data_type_updated = 0
```

Comment:
I made a mistake with the variable name in https://github.com/z3z1ma/dbt-osmosis/pull/98 😇

Reply:
Ah my bad for not catching either